inst/api/cemconnector_openapi.yaml

```diff
@@ -0,0 +1,350 @@
+servers:
```

Comment:
How is this file generated? I'd like to add it to package maintenance so this is updated with each release version

Reply:
Unfortunately plumber doesn't support generating an openapi schema, so this file wasn't generated. It was created manually and can be maintained by importing it into the online editor here:

https://editor.swagger.io/

Defining an openapi schema adds a lot of value to the API user. With a schema defined, the generated 'try it' form provides a template of the input JSON args (otherwise it is just a blank text box) and the output JSON args are also specified clearly which is helpful for the folks that need to integrate their systems with the API.

Reply:
Ahh ok, this is good to know as I'll have to maintain this as part of the releases. I agree, it adds a lot of value.